river/InputManager.zig

```diff
@@ -43,6 +95,8 @@ relative_pointer_manager: *wlr.RelativePointerManagerV1,
 virtual_pointer_manager: *wlr.VirtualPointerManagerV1,
 virtual_keyboard_manager: *wlr.VirtualKeyboardManagerV1,
 
+input_configs: std.SinglyLinkedList(InputConfig) = .{},
```

Comment:
We don't seem to store pointers to `InputConfig`s anywhere which would require pointer stability. Therefore, I think we could just use an `ArrayList` (or `ArrayListUnmanaged`) here which should simplify things a bit.